string robot_name
time runtime
bool run_before
time[] arival_times
uint8[] health_at_goal
geometry_msgs/Point[] given_points
geometry_msgs/Point[] expected_points
